sample/src/main/kotlin/com/airbnb/lottie/samples/utils/TypeExtensions.kt

```diff
@@ -63,7 +63,7 @@ inline fun <reified T> flatten(vararg lists: List<T>?) = lists.flatMap { it ?: e
 
 fun Float.lerp(other: Float, amount: Float): Float = this + amount * (other - this)
 
-fun Float.sqrt() = Math.sqrt(this.toDouble()).toFloat()
+fun Float.sqrt() = kotlin.math.sqrt(this.toDouble()).toFloat()
```

Comment:
You can import kotlin.math.sqrt

Reply:
Import this will lead to a recursive problem
`Type checking has run into a recursive problem. Easiest workaround: specify types of your declarations explicitly`